### PR TITLE
docs(common):  Updated readme ENG #370

### LINKED ISSUE #370: Fix Readme
- [x] Logo Size
- [x] Version Note
- [x] screenshot

### PR DESCRIPTION
**Pull request by fvives99**
Changed logo for it to be visible in dark mode, updated developers guide screenshot, and removed version note.

### What does this PR do?

- resolves #370 and #485 

### Steps to test
1. Open guides readme, see new screenshot, updated logo, and updated prerequisites.

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,8 +120,8 @@ If you find a bug, please report big and small bugs by [opening an issue](https:
 
 <p align="center">
 	<a href="https://eoscostarica.io/">
-		<img src="https://raw.githubusercontent.com/eoscostarica/design-assets/master/logos/eosCR/fullColor-horizontal-transparent-white.png"
-		width="400px" >
+		<img src="https://user-images.githubusercontent.com/5632966/178800859-e05a07ff-b3e5-41a9-9137-d1a271b958ff.png"
```
Comment (xavier506) [inline README.md:123]: please update image URL to this one: https://raw.githubusercontent.com/eoscostarica/.github/master/.github/workflows/images/eos-costa-rica-logo.png

Comment (fvives99) [reply README.md]: Done.

**Comment by xavier506** [comment]
The screenshot seems way too large for the readme , also maybe we should we wait for the new version of the homepage to add an updated screenshot?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
-# EOS COSTA RICA DEVELOPER GUIDES
+# EOS Costa Rica Developer Guides
 ![](https://img.shields.io/github/license/eoscostarica/guias.eoscostarica.io) ![](https://img.shields.io/badge/code%20style-standard-brightgreen.svg) ![](https://img.shields.io/badge/%E2%9C%93-collaborative_etiquette-brightgreen.svg) ![](https://img.shields.io/twitter/follow/eoscostarica.svg?style=social&logo=twitter) ![](https://img.shields.io/github/forks/eoscostarica/guias.eoscostarica.io?style=social)
 
+
+
 EOS Costa Rica Developers Documentation Site.
 
-![Guide front page](static/img/guide-page.png)
+![Guide front page](https://user-images.githubusercontent.com/55892352/179574354-73a0d8de-301f-448b-a2f8-e6bf54f6eb6b.png)
```
Comment (xavier506) [inline README.md:8]: Lets save this image inside `static/img/`

**Review by xavier506 — CHANGES_REQUESTED** [review]
see comments

File: README.md (hunk 2 above)
Comment (fvives99) [reply README.md]: fixed, sorry for the delay, didn't see the comment

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**